packages/node/test/integration.test.js

```diff
@@ -32,6 +32,9 @@ const testsThatFailToBuild = new Map([
 
 // eslint-disable-next-line no-restricted-syntax
 for (const fixture of fs.readdirSync(fixturesPath)) {
+  if (fixture != '05-ts-declaration-asset') {
```

Comment:
Remove?

Reply:
Oops, didn't mean to commit that. Fixed in [23f87f4](https://github.com/vercel/vercel/pull/7918/commits/23f87f46bbceb37f9ea12dfc92719b11b67b02af)